tests/Tracing/GuzzleTracingMiddlewareTest.php

```diff
@@ -104,7 +104,8 @@ public function testTrace(Request $request, $expectedPromiseResult, array $expec
         $hub->setSpan($transaction);
 
         $middleware = GuzzleTracingMiddleware::trace($hub);
-        $function = $middleware(static function () use ($expectedPromiseResult): PromiseInterface {
+        $function = $middleware(static function (Request $request) use ($expectedPromiseResult): PromiseInterface {
+            self::assertNotEmpty($request->getHeader('sentry-trace'));
```

Comment:
Just to keep consistency with how we write assertions usually, I think it would be better to invoke the `assertNotEmpty()` method on `$this`

```suggestion
        $function = $middleware(function (Request $request) use ($expectedPromiseResult): PromiseInterface {
            $this->assertNotEmpty($request->getHeader('sentry-trace'));
            
```